app/controllers/floating_ip_controller.rb

```diff
@@ -128,15 +128,15 @@ def edit
     drop_breadcrumb(
       :name => _("Associate Floating IP \"%{address}\"") % { :address => @floating_ip.address },
       :url  => "/floating_ip/edit/#{@floating_ip.id}")
-    render :change
+    render :new_edit
```

Comment:
This works but is inconsistent with usual way it's done. It should be `render :action => "floating_ips/new_edit"` or `render :template => "floating_ips/new_edit"`. So my main concern is consistency :)

Reply:
Sure. Done

Reply:
I disagree with this, the RoR controller provides the context this is why there is no need for absolute path and as a matter of fact, there is no other example across the GUI besides the special case of "layouts". So unless there is a new "practice" to be set, I'm keeping it this way. On top of that, as I already mentioned before, this doesn't matter since the component approach takes over the rendering.

Reply:
I agree that it works :) but it's **never** used this way in the project (except once by you). Common practice here would be to have `new` and `edit` templates (that will be rendered automatically) that have something like
``` haml
#main_div
  = render :partial => 'floating_ip_form'
```
and rename `new_edit` to `_floating_ip_form`.

Or if that is not an option then `render :action => "new_edit"` or `render :template => "new_edit"` (error in first comment) to be consistent with practice in this project.

Please choose one :)